backend/webdav/webdav.go

```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/rclone/rclone/lib/pacer"
 	"github.com/rclone/rclone/lib/rest"
 
+	"github.com/icholy/digest"
```

Comment:
Linting error here, from the goimports tool. Think it needs to be moved up to just before the first "github.com/rclone" (sorted order).

Reply:
I've just moved it up to where it belongs.

Go's really not my thing, so I thought this belonged below, along with ntlmss (another external import)